data/sql/updates/pending_db_world/rev_1724722289062405500.sql

```diff
@@ -0,0 +1,4 @@
+--
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_gen_select_target_count_15_1' AND `spell_id` = 41357;
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(41357, 'spell_gen_select_target_count_15_3');
```

Comment:
Delete query and insert query do not match

Reply:
And they shouldn’t, because we are deleting a preexisting entry. But sure, we don’t need to specify the preexisting script name here. I was just being extra careful.